### PR TITLE
added scripts and data retrieved from unicode CLDR

### PR DESCRIPTION
**Pull request by sarthak5**
Added scripts use to retrieve and store data in desired format, and added the data retrieved. Still support for numbering systems and numerals need to be added and some more issues are to be dealt with.

### CODE REVIEW

File: scripts/get_cldr_data.py [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+
+    try:
+        date_format_string = gregorian_dict["dateFormats"]["short"].upper()
+    except:
+        date_format_string = gregorian_dict["dateFormats"]["short"]["_value"].upper()
+
+    json_dict["date_order"] = DATE_ORDER_PATTERN.sub(
+            r'\1\2\3', DATE_ORDER_PATTERN.search(date_format_string).group())
+
+    json_dict["january"] = [gregorian_dict["months"]["stand-alone"]["wide"]["1"],
```
Comment (asadurski) [inline scripts/get_cldr_data.py]: This dict creation part could be made a lot shorter with loops.

File: scripts/get_cldr_data.py [hunk 2]
```diff
@@ -0,0 +1,480 @@
+import requests
```
Comment (sarthak5) [inline scripts/get_cldr_data.py]: But these are imported in the scripts which won't be used by users, do we still need to add these to requirements?

File: scripts/get_cldr_data.py [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+import time
+from collections import OrderedDict
+from orderedset import OrderedSet
+import base64
+import six
+
+from utils import get_dict_difference
+from order_languages import language_locale_dict
+
+OAuth_Access_Token = 'OAuth_Access_Token'       # Add OAuth_Access_Token here
```
Comment (sarthak5) [inline scripts/get_cldr_data.py]: I didn't know about that, will make changes soon.

**Comment by sarthak5** [comment]
Thanks for the review @asadurski. I have modified the scripts and they work much faster now.

**Pull request merged**